core/server/web/middleware/image/normalize.js

```diff
@@ -7,7 +7,11 @@ const image = require('../../../lib/image');
 module.exports = function normalize(req, res, next) {
     const imageOptimizationOptions = config.get('imageOptimization');
 
-    if (!imageOptimizationOptions.resize) {
+    // NOTE: .gif optimization is currently not supported by sharp but will be soon
+    //       as there has been support added in underlying libvips library https://github.com/lovell/sharp/issues/1372
+    //       As for .svg files, sharp only supports conversion to png, and this does not
+    //       play well with animated svg files
+    if (!imageOptimizationOptions.resize || ['.gif', '.svg', '.svgz'].includes(req.file.ext)) {
```

Comment:
LGTM 👍 Wanne add one unit test? 🙊

Reply:
@kirrg001 unit tests added and some cleanup done :blush: